opentelemetry-otlp/tests/integration_test/tests/logs.rs

```diff
@@ -130,7 +138,38 @@ mod logtests {
         let logger_provider = rt.block_on(async {
             // While we're here setup our collector container too, as this needs tokio to run
             test_utils::start_collector_container().await?;
-            init_logs()
+            init_logs(false)
+        })?;
+        let layer = layer::OpenTelemetryTracingBridge::new(&logger_provider);
+        let subscriber = tracing_subscriber::registry().with(layer);
+        // generate a random uuid and store it to expected guid
+        let expected_uuid = Uuid::new_v4().to_string();
+        {
+            let _guard = tracing::subscriber::set_default(subscriber);
+            info!(target: "my-target",  uuid = expected_uuid, "hello from {}. My price is {}.", "banana", 2.99);
+        }
+
+        let _ = logger_provider.shutdown();
+        std::thread::sleep(Duration::from_secs(5));
+        assert_logs_results(test_utils::LOGS_FILE, expected_uuid.as_str())?;
+        Ok(())
+    }
+
+    #[test]
+    #[cfg(any(feature = "tonic-client", feature = "reqwest-blocking-client"))]
+    pub fn logs_simple_non_tokio_main() -> Result<()> {
+        logs_simple_non_tokio_helper()
+    }
+
+    fn logs_simple_non_tokio_helper() -> Result<()> {
+        // Initialize the logger provider inside a tokio runtime
+        // as this allows tonic client to capture the runtime,
```

Comment:
for reqwest-blocking, this can work without outside tokio too, will add a test to cover that also in a follow up PR.

Reply:
yes, I added similar comment for metrics too - https://github.com/open-telemetry/opentelemetry-rust/pull/2455#discussion_r1891538909